frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh

```diff
@@ -50,7 +54,14 @@ EOF
 
 # Install virtctl binary and add in to PATH
 mkdir virtctl
-wget https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/virtctl-${KUBEVIRT_VERSION}-linux-x86_64 -O virtctl/virtctl
+
+url1=$(curl "${VIRTCTL_X86_64}")
+url2=$(curl "${VIRTCTL_AMD64}")
+
+[ "${url1}" != "Not Found" ] && wget ${VIRTCTL_X86_64} -O virtctl/virtctl
```

Comment:
there is no need to do curl first - let's just do wget and check the return code of that. Theoretically you could just do 

`wget ${VIRTCTL_AMD64} -O virtctl/virtctl || wget ${VIRTCTL_X86_64} -O virtctl/virtctl`